src/Paramore.Brighter.Outbox.DynamoDB/DynamoDbOutbox.cs

```diff
@@ -182,16 +182,11 @@ public async Task<Message> GetAsync(Guid messageId, int outBoxTimeout = -1, Canc
             return messages;
         }
 
-        /// <summary>
-        /// Get paginated list of Messages.
-        /// </summary>
-        /// <param name="pageSize"></param>
-        /// <param name="pageNumber"></param>
-        /// <returns>A list of messages</returns>
-        public IList<Message> Get(
-            int pageSize = 100, 
-            int pageNumber = 1, 
-            Dictionary<string, object> args = null)
+        [Obsolete("Removed in v10, Please use OutstandingMessages instead.")]
+        IList<Message> IAmAnOutboxSync<Message>.Get(
```

Comment:
Making not supported methods as explicit interface methods to hide them from direct usage. Although in all likelihood they will be used via the interface anyway, just a good practice